RULES.md

```diff
@@ -139,6 +139,7 @@ Each Notice is associated with a severity: `INFO`, `WARNING`, `ERROR`.
 | [`unusable_trip`](#unusable_trip)                                                             | Trips must have more than one stop to be usable.                                                                                                              |
 | [`unused_shape`](#unused_shape)                                                               | Shape is not used in GTFS file `trips.txt`.                                                                                                                   |
 | [`unused_trip`](#unused_trip)                                                                 | Trip is not be used in `stop_times.txt`                                                                                                                       |
+| [`missing_bike_allowance_info`](#missing_bike_allowance_info)                                 | `bikes_allowed` value should be specified for ferry trips                                                                                                     |
```

Comment:
Don't you think that having _info at the end could lead to confusion since this is actually a warning?

Reply:
Agreed. I meant by this name "the bike allowance information is missing". What name would you suggest?

Reply:
suggestion: what about if we remove the suffix and leave it as: `missing_bike_allowance`

** Please merge master to this branch as per #1517; We are not updating RULES.md anymore. The name changes need to be done only in the Notice Java class name.